packages/server/src/admin/super.ts

```diff
@@ -221,6 +222,41 @@ superAdminRouter.post(
   })
 );
 
+// POST to /admin/super/dbstats
+// to query database statistics.
+superAdminRouter.post(
+  '/dbstats',
+  asyncWrap(async (req: Request, res: Response) => {
+    requireSuperAdmin();
+    requireAsync(req);
+
+    await sendAsyncResponse(req, res, async () => {
+      const systemRepo = getSystemRepo();
+      const client = getDatabasePool();
+      const sql = `
+      SELECT * FROM (
+        SELECT table_schema, table_name, pg_relation_size('"'||table_schema||'"."'||table_name||'"') AS table_size
+        FROM information_schema.tables
+      ) tables
+      WHERE table_size > 0
+      ORDER BY table_size DESC;
+      `;
+
+      const result = await client.query(sql);
+
+      const contentString = result.rows
+        .map((row) => `${row.table_schema}.${row.table_name}: ${row.table_size}`)
+        .join('\n');
+
+      await systemRepo.createResource<Communication>({
+        resourceType: 'Communication',
```

Comment:
Why `Communication` here, and why is it persisted to the DB?

Reply:
With the existing `/admin/super/x` APIs, they all require async, and don't have any representation of return value.

It's a good question: how should we capture the return value of a async operation?  We could add the output `Parameters` to the `AsyncJob`?  It would still be persisted in the db, but at least it wouldn't be as contrived as `Communication`.

Reply:
Yeah, that would be my suggestion — persisted `Parameters` sounds good to me

Reply:
You broke the one rule!
https://hl7.org/fhir/R4/parameters.html#:~:text=Scope%20and%20Usage-,This%20resource%20is%20a%20non%2Dpersisted%20resource,-used%20to%20pass

But I mean yeah I think it makes the most sense